### DISCUSSION TITLE
Disable Audio of the video

### DISCUSSION THREAD

**Discussion by Manuelagerstl** [olivierfriard/BORIS#571] [Q&A]
Hi,
i want to overlap an audiofile (where there is just audio, no video) and a videofile(which contains an audio and a videofile). As the audio of the videofile contains a lot of background noise I want to mute it, but I can not find a button to do it. When i want to edit it in the observation it is not possible to do that either. 
Can anybody help me?
Thx
Manuela

**Comment by olivierfriard** [comment]
Hi,
each media player has aa audio volume control on its right.
See https://boris.readthedocs.io/en/latest/#the-media-player-widgets for details

**Reply by Manuelagerstl** [reply]
Thank you so much for your fast help!
